### PR TITLE
Issue #1202 Changing the default version of OpenShift to v3.6.0

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: cmd/minishift/cmd/start_test.go [hunk 1]
```diff
@@ -131,9 +131,9 @@ func (r *RecordingRunner) Output(command string, args ...string) ([]byte, error)
 
 var (
 	testConfig = &clusterup.ClusterUpConfig{
-		OpenShiftVersion: "v1.5.1",
+		OpenShiftVersion: "v3.6.0",
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start_test.go]: Changed the version here as it is drastic change in naming pattern from Origin.

**Comment by LalatenduMohanty** [comment]
Looks like integration tests are failing for coolstore

```
09:11:28 
09:11:28 --- Failed scenarios:
09:11:28 
09:11:28     features/basic.feature:117
09:11:28 
09:11:28 26 scenarios (25 passed, 1 failed)
09:11:28 109 steps (108 passed, 1 failed)
```

**Comment by hferentschik** [comment]
> Looks like integration tests are failing for coolstore

coolstrore is or should not be tested per default. There must be something else going on. Message seems indeed a bit weird.

**Comment by hferentschik** [comment]
retest this please

**Comment by hferentschik** [comment]
Hmm, coolstore might be running indeed per default. I don't think this is intentional. AFAIR the plan was to run coolstore only on a nightly basis, since it takes such a long time to run. @gbraad, wdyt?

**Comment by hferentschik** [comment]
Looking at error message more closely, it seems the persistent volume setup/check failed:

    features/basic.feature:117

**Comment by hferentschik** [comment]
There is a name change. It is now 'persistentvolumes/pv0001', our assertion is for 'persistentvolume/pv0001'. Hence the test failure.

**Comment by gbraad** [comment]
Coolstore does not run... Likely what you are seeing is the following:

```
Feature: Coolstore
   blah bla

Feature: Basic
   blah bla

  Scenario: blah bla
     Given ...
```

It will always show the featured that are available, but it only runs the tags we request for.

**Comment by agajdosi** [comment]
> It will always show the featured that are available, but it only runs the tags we request for.

We can get over this by specifying tests not by tags but by filenames.

**Comment by gbraad** [comment]
I think we should be able to run multiple tests, but the confusioon came from how godog reports this back.

I did not find a scenario that matched the criteria. It does however showo the features it has tried to loook for this... nothing wrong ;-)

**Comment by hferentschik** [comment]
@gbraad 

> Coolstore does not run... Likely what you are seeing is the following:

It does. As mentioned in another comment, this is not the issue and it is also not failing on coolstore test. The problem is with the persistent volume check. I am testing an update as we speak.

Regarding coolstore:

    $ make integration 

does run it. 

Look at the target:

    go test -timeout 3600s $(REPOPATH)/test/integration --tags=integration -v -args --test-dir $(INTEGRATION_TEST_DIR) --binary $(MINISHIFT_BINARY) $(GODOG_OPTS)

What would exclude it from being run? 

So I think we should open a new issue to actually make sure that coolstore is not run per default and maybe while on it setup the CI job to run the full integration test suite as part of a nightly build.

**Comment by gbraad** [comment]
CentOS CI runs the command as mentioned above:

```
$ make integration GODOG_OPTS="-tags basic -format pretty"
```

**Comment by hferentschik** [comment]
@gbraad I was about to take it back myself ;-) I've seen this as well now. I would argue though we should change the Makefile as well.

**Comment by gbraad** [comment]
I think we need to be able to specify the tags as follows:

```
$ make integration basic
```

**Comment by hferentschik** [comment]
Updated Cucumber test and merged.

**Comment by agajdosi** [comment]
@gbraad +1 for changing the tag to `@nightly` and adding it into basic.feature also. We can later go for several tiers describing time cost of tests: basic aka quick, nightly and complete/release.

Please create an issue for `make integration basic` will be happy to implement it. We had a discussion around this already I think, but back then we agreed that we will try it with GODOG_OPTS, however `make integration basic` and probably also `make integration nightly` are just friendlier.

**Pull request closed without merging**